src/main/kotlin/com/terraformation/backend/customer/AppNotificationService.kt

```diff
@@ -337,6 +342,26 @@ class AppNotificationService(
     }
   }
 
+  @EventListener
+  fun on(event: ModuleEventStartingEvent) {
+    systemUser.run {
+      val moduleEvent = moduleEventStore.fetchOneById(event.eventId)
+      val renderMessage = {
+        messages.moduleEventStartingNotification(
+            parentStore.getEventType(event.eventId)!!,
```

Comment:
Could the event type live in `EventModel`? Then we don't need to fetch it separately.